R/Rules-class.R

```diff
@@ -675,6 +675,110 @@ NextBestMaxGainSamples <- function(prob_target_drt,
   )
 }
 
+# NextBestProbMTDLTE ----
+
+## class ----
+
+#' `NextBestProbMTDLTE`
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' [`NextBestProbMTDLTE`] is the class of finding a next best dose that selects
+#' the dose with the highest probability of having a toxicity rate less or equal
+#' to the toxicity target.
+#' The dose is determined by calculating the posterior toxicity probability
+#' for each dose per iteration and select the maximum dose that has a toxicity
+#' probability below or equal to the target. The dose with the highest frequency
+#' of being selected as MTD across iterations is the next best dose. Placebo
+#' is not considered in the calculation and removed from the dose grid for
+#' any calculations.
```

Comment:
this description is not precise enough. So we either keep it generic, or better specified.

Reply:
Are referring to the placebo part? Or to the whole description? Will add some background why placebo is removed.